Src/VimCore/CommandUtil.fs

```diff
@@ -1084,6 +1086,22 @@ type internal CommandUtil
         let span = action snapshotData
         BufferGraphUtil.MapSpanDownToSingle _bufferGraph span x.CurrentSnapshot
 
+    /// Extend the selection for a mouse click
+    member x.ExtendSelectionForMouseDrag (visualSpan: VisualSpan) =
+
+        // Double-clicking creates a problem because the caret is moved to the
+        // end of what was selected rather than directly under the mouse
+        // pointer.  Prevent accidentally dragging after a double-click by
+        // moving the caret only if the mouse position is over a different
+        // snapshot point than it was when the mouse was previously clicked.
+        x.MoveCaretToMouseIfChanged() |> ignore
+        CommandResult.Completed ModeSwitch.NoSwitch
+
+    /// Extend the selection for a mouse drag
+    member x.ExtendSelectionForMouseClick (visualSpan: VisualSpan) =

```

Comment:
>(visualSpan: VisualSpan) [](start = 42, length = 24)

Why does this take a VisualSpan and not use it?

Reply:
Will remove.